benchopt/cli.py

```diff
@@ -152,7 +155,9 @@ def run(benchmark, solver_names, forced_solvers, dataset_names,
               "folder.")
 @click.option('--kind', '-k', 'kinds',
               multiple=True, show_default=True, type=str,
-              help='Timeout a solver when run for more than <timeout> seconds')
+              help="Specify the type of figure to plot: " +
+              ", ".join("`{0}`".format(k) for k in PLOT_KINDS_NAMES[:-1]) +
```

Comment:
Use f-string when possible:
```suggestion
              ", ".join(f"`{k}`" for k in PLOT_KINDS_NAMES[:-1]) +
```